sync/sync.md

```diff
@@ -0,0 +1,95 @@
+# Sync
+
+Syncer implements efficient synchronization for headers.
+
+There are two main processes running in Syncer:
+
+* Main syncing loop(`syncLoop`)
+  * Performs syncing from the latest stored header up to the latest known Subjective Head
+    * Subjective head: the latest known local valid header and a sync target.
+    * Network head: the latest valid network-wide header. Becomes subjective once applied locally.
+  * Syncs by requesting missing headers from Exchange or
+  * By accessing cache of pending headers
+* Receives every new Network Head from PubSub gossip subnetwork (`incomingNetworkHead`)
+  * Validates against the latest known Subjective Head, is so
+  * Sets as the new Subjective Head, which
+  * If there is a gap between the previous and the new Subjective Head
+  * Triggers s.syncLoop and saves the Subjective Head in the pending so s.syncLoop can access it
+
+For creating a new instance of the Syncer following components are needed:
+
+* A getter, e.g., [Exchange][exchange]
+* A [Store](../../../store/store.md)
```

Comment:
use absolute link instead, looks like this is a relative path from a previous location